client/context.go

```diff
@@ -49,9 +50,10 @@ type Context struct {
 	FeePayer          sdk.AccAddress
 	FeeGranter        sdk.AccAddress
 	Viper             *viper.Viper
-	
+	GRPC              grpc.ClientConnInterface
```

Comment:
```suggestion
	ClientConn              grpc.ClientConnInterface
```

I have a small preference for calling this `ClientConn`, so that we could create a TM-rpc-based ClientConn and pass it here too. But not a deal breaker.

Reply:
makes sense